addons/repair/stringtable.xml

```diff
@@ -502,7 +502,7 @@
             <Chinese>完整維修部分</Chinese>
         </Key>
         <Key ID="STR_ACE_Repair_RepairedPartially">
-            <English>Partially repaired %1</English>
+            <English>Partially repaired part</English>
             <German>Bauteil zum Teil repariert</German>
             <Polish>Częściowo naprawiono: %1</Polish>
```

Comment:
Several of the translations have %1 in them as well. Remove those translations?

Reply:
updated them all, not sure why German was already fixed